src/components/PostDetails.tsx

```diff
@@ -1,117 +1,127 @@
 import React from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { Error } from '../types/Error';
+import { deleteComment } from '../api';
+
+type Props = {
+  post?: Post,
+  isLoading: string,
+  comments: Comment[],
+  onAddComment: (newComment: Comment) => void,
+  onDeleteComment: (comments: Comment[]) => void,
+  isError: Error,
+  formIsActive: boolean,
+  onSetFormIsActive: (arg: boolean) => void,
+};
+
+export const PostDetails: React.FC<Props> = ({
+  post,
+  isLoading,
+  comments,
+  isError,
+  formIsActive,
+  onAddComment,
+  onDeleteComment,
+  onSetFormIsActive,
+}) => {
+  const handleCommentDelete = (commentId: number) => {
+    const filteredComments
+      = comments.filter(comment => comment.id !== commentId);
+
+    onDeleteComment(filteredComments);
+    deleteComment(commentId)
+      .then()
```

Comment:
You should handle errors and show appropriate notifications after calling deleteComment function. Add a catch block with proper error handling.